src/components/Greetings.js

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { retrieveGreetings } from '../app/greetings';
+
+function Greeting() {
+  const greetings = useSelector((state) => state.greetings);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(retrieveGreetings());
+  }, []);
+
+  return (
+    <div>
+      {greetings.map((greeting) => (
```

Comment:
## Kindly display random greetings according to the [project requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/hello_world_two_apps.md#:~:text=Display%20the%20random%20greeting%20in%20your%20Greeting%20component.) as currently, you are rendering all the greetings at once, bellow  👇 are screenshots of my suggestions on fixing these issues  
![image](https://user-images.githubusercontent.com/52497246/221358328-68fbf3e9-92b2-455b-84c3-59d1292d21c7.png)

![image](https://user-images.githubusercontent.com/52497246/221358320-fead1b8a-f754-40eb-b9f9-e24b256474f6.png)

![image](https://user-images.githubusercontent.com/52497246/221358250-1d7ec248-6753-485a-a9ef-c5e58c1b5afd.png)